### PR TITLE
Range updates & scheduling fixes

### PR DESCRIPTION
**Pull request by emerencia**
## In this PR:

- Enables ticks on sliders.
- Fixes OTR bugs with scheduled protocols.
- Fixes that protocols could be started in the past, resulting in no responses being scheduled.
- Added translation for Sunday questionnaire

Large screen size:

chrome_2020-10-19_19-58-30

Small screen size:

chrome_2020-10-19_19-57-54

### CODE REVIEW

File: app/controllers/api/v1/basic_auth_api/protocol_subscriptions_controller.rb [hunk 1]
```diff
@@ -49,13 +49,17 @@ def set_protocol_subscription
...
-          Time.zone.parse(protocol_subscription_create_params[:start_date])
+          # The start date cannot be in the past.
+          [Time.zone.parse(protocol_subscription_create_params[:start_date]), Time.zone.now].max
         end
 
         def end_date
           return nil if protocol_subscription_create_params[:end_date].blank?
 
-          Time.zone.parse(protocol_subscription_create_params[:end_date])
+          # The duration between start and end date should be at least one hour.
```
Comment (frbl) [inline app/controllers/api/v1/basic_auth_api/protocol_subscriptions_controller.rb:59]: why is this? Can you add a comment why it should be at least 1 hour?

**Review by frbl — APPROVED** [review]
lgtm

File: app/controllers/api/v1/basic_auth_api/protocol_subscriptions_controller.rb (hunk 1 above)
Comment (emerencia) [reply app/controllers/api/v1/basic_auth_api/protocol_subscriptions_controller.rb:59]: So people have at least one hour to fill out the measurement.

Comment (emerencia) [reply app/controllers/api/v1/basic_auth_api/protocol_subscriptions_controller.rb:59]: I added a comment.

**Pull request merged**